.php-cs-fixer.dist.php

```diff
@@ -62,12 +62,21 @@
         'numeric_literal_separator' => true,
         'phpdoc_order' => [
             'order' => [
+                'deprecated',
+                'final',
+                'readonly',
+                'internal',
+                'no-named-arguments',
```

Comment:
These 5 tags are important for the possible usages of the class, so I would place them at the top.